test/projectile-test.el

```diff
@@ -98,3 +98,10 @@ test temp directory"
           (projectile-known-projects-file (projectile-test-tmp-file-path)))
       (projectile-clear-known-projects)
       (expect projectile-known-projects :to-equal nil))))
+
+(describe "projectile-test-ignored-directory-p"
+  (it "ignores specified directories"
```

Comment:
I think you can have a separate `if` for the regexp check to highlight it.

Reply:
Good call, I added a separate spec for it